builder/dockerfile/internals.go

```diff
@@ -114,8 +115,8 @@ func (b *Builder) commitContainer(dispatchState *dispatchState, id string, conta
 	return err
 }
 
-func (b *Builder) exportImage(state *dispatchState, imageMount *imageMount, runConfig *container.Config) error {
-	newLayer, err := imageMount.Layer().Commit()
+func (b *Builder) exportImage(state *dispatchState, layer builder.RWLayer, parent *imageMount, runConfig *container.Config) error {
```

Comment:
only `parent.Image()` is used, so maybe accept `parent *image.Image` instead of `imageMount`?

Reply:
replaced with `builder.Image`